src/oogeso/core/networks/network_node.py

```diff
@@ -162,80 +145,79 @@ def _rule_pressureAtNode(self, model, carrier, t):
             return pyo.Constraint.Skip
         else:
             # single terminal. (pressure out=pressure in)
-            expr = (
-                model.varPressure[(node, carrier, "out", t)]
-                == model.varPressure[(node, carrier, "in", t)]
-            )
+            expr = model.varPressure[(node, carrier, "out", t)] == model.varPressure[(node, carrier, "in", t)]
             return expr
 
-    def _rule_pressureBounds(self, model, term, carrier, t):
+    def _rule_pressure_bounds(
+        self, model: pyo.Model, term, carrier, t: int
+    ) -> Union[bool, pyo.Expression, pyo.Constraint.Skip]:
         node = self.id
         nominal_pressure = self.pressure_nominal
-        maxdev = None  # default is no constraint
+        max_dev = None  # default is no constraint
         if carrier in nominal_pressure:
             if term in nominal_pressure[carrier]:
                 nom_p = nominal_pressure[carrier][term]
                 if nom_p is not None:
-                    if (carrier in self.pressure_maxdeviation) and (
-                        term in self.pressure_maxdeviation[carrier]
-                    ):
-                        maxdev = self.pressure_maxdeviation[carrier][term]
+                    if (carrier in self.pressure_max_deviation) and (term in self.pressure_max_deviation[carrier]):
+                        max_dev = self.pressure_max_deviation[carrier][term]
                         if t == 0:
                             logger.debug(
                                 "Using individual pressure limit for: {}, {}, {}, {}".format(
-                                    node, carrier, term, maxdev
+                                    node, carrier, term, max_dev
                                 )
                             )
                     else:
                         # Using globally set pressure deviation limit
-                        maxdev = model.paramMaxPressureDeviation
-        if (maxdev is None) or (maxdev == -1):
+                        max_dev = model.paramMaxPressureDeviation
+            else:
+                nom_p = 0
+                # raise ValueError("nom_p has no value")  # Fixme: How do we want to handle this case? nom_p unresolved.
```

Comment:
Should skip constraint (max_dev=None)

I don't think the else statements are needed as max_dev=None by default. 
Could  use return pyo.Constraint.Skip to make it more explicit